app/views/components/summaryList.scala.html

```diff
@@ -35,7 +37,11 @@
             <a class="govuk-link" href="@summary.rows(fullstop).actions.head.items.head.href">
                 <span class="govuk-visually-hidden">
                     @summary.rows(fullstop).actions.head.items.head.visuallyHiddenText</span>
-                <span> @Html(messages("cf.cash-account.detail.csv.download"))</span>
+                <span>
+                    @Html(messages(
+                        if (change) "cf.cash-account.transactions.request.review.change"
```

Comment:
Could you please write the tests for this change? If it takes time then leave it for now.